### PR TITLE
Added CAN Information Document

### PR DESCRIPTION
**Pull request by safasyed**
- contains information on what CAN is, CAN protocol and how it works

### CODE REVIEW

File: Documentation/CAN.adoc [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (bill-luu) [inline Documentation/CAN.adoc]: Looots of trailing whitespace here

Comment (adolapo) [reply Documentation/CAN.adoc]: Saammmeee

File: Documentation/CAN.adoc [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+* A CAN message is made up of 10 bytes of data and is sent in a format called a frame 
+* A frame has a defined structure and carries a meaningful sequence of bits or bytes of data
+* There are two types of frames:
+    - standard: 11 bit identifier fields (Solar Car uses this)
+image:References/11bitCAN.jpg[]
+    - exetended: 29 bit identifier fields
+* All nodes on CAN receive the CAN frame and depending on the ID on the node CAN decides whether to accept it or not
+* If multiple nodes send the message at the same time then the node with the highest priority (lowest arbitration ID) will get bus access and low priroty nodes wait until the bus is available
+
+Note: The CAN network needs two terminating resistors in order to work
```
Comment (bill-luu) [inline Documentation/CAN.adoc]: Also add in the actual resistance needed

File: Documentation/CAN.adoc [hunk 3]
```diff
@@ -33,4 +33,4 @@ image:References/11bitCAN.jpg[]
 * All nodes on CAN receive the CAN frame and depending on the ID on the node CAN decides whether to accept it or not
 * If multiple nodes send the message at the same time then the node with the highest priority (lowest arbitration ID) will get bus access and low priroty nodes wait until the bus is available
 
-Note: The CAN network needs two terminating resistors in order to work
+Note: The CAN network needs two terminating resistors in order to work - 2 120ohm resistors in parallel (A total of 60 ohms)
```
Comment (bill-luu) [inline Documentation/CAN.adoc]: Do `two 120ohm resistors` instead.

**Pull request merged**